src/views/router_views/email/change-email-address.njk

```diff
@@ -8,7 +8,7 @@
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 
-{% set title = "What is the new registered email address?" %}
+{% set title = "New registered email address" %}
```

Comment:
The prototype has the following:
![image](https://github.com/companieshouse/registered-email-address-web/assets/123395274/acf90dbf-ddf3-4fb0-89f2-818deb33b6e0)

Does this need to change?

Reply:
title actually refers to the text we see in the tab not the page title but its used for tracking events in matomo which is why it was updated.
but good spot I updated this and all the titles we use to match the prototype 
![Uploading Screenshot 2023-07-12 at 14.49.15.png…]()